ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -471,3 +471,13 @@ def __setitem__(self, key, value):
 
     def __mod__(self, value, /):
         return np_frontend.mod(self._ivy_array, value, out=self)
+    
+    def ptp(self, *, axis=None, dtype=None, out=None, keepdims=False, where=True):
```

Comment:
The [original method](https://numpy.org/doc/stable/reference/generated/numpy.ptp.html) does not take `dtype` as an argument. Hence, it's fine to remote it from here.